src/cmd-import

```diff
@@ -150,9 +176,12 @@ def generate_lockfile_and_commitmeta(tmpd, ostree_commit, build_meta):
     with open(tmp_commitmeta, 'w') as f:
         json.dump(commitmeta, f, indent=2)
 
+    iso_format = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
For some reason. (I'm not sure entirely of the history), but the timestamp was always `T00:00:00Z` so nothing more specific than just the day. we might have to dig into the rpm-ostree code to understand why, but absent of doing that and discounting the reason, I think we should keep the same semantics here.